lib/sequent/core/command_service.rb

```diff
@@ -57,6 +46,31 @@ def remove_event_handler(clazz)
 
       private
 
+      def process_commands
```

Comment:
Can we extract this to a separate class like the EventPublisher. Both are quite similar in intent and behavior. Same suggestion made for the EventPublisher also apply here then :-)

Reply:
`CommandService` is a separate class like `EventPublisher` already.